documentation/hazards/Hazards.tex

```diff
@@ -128,23 +130,18 @@ \subsubsection*{Mitigation}
 \subsubsection*{Safety Requirements}
 Section 11.9 of our System Requirements Rev0 document covers this hazard.
 
-\subsection{User's Password Gets Leaked}
+\subsection{User's Magic Link Gets Leaked}
 
-\emph{Since we are no longer using a password-based system, we should rewrite this section}
-\begin{comment}
-\subsubsection*{Explanation}
-Depending on how the user stores their password (written down or in a file on the computer), an attacker may attain the user's password through various means. Ergo, the attacker may impersonate the user on Book Bazar. A worst-case-scenario would be if the user uses the same password for multiple services and the attacker could now access all these other services using the user's password.
 
+\subsubsection*{Explanation}
+To verify that a user is a member of the McMaster community, Book Bazar shall generate magic links which will be emailed to their McMaster emails. There is a risk of a malicious user either guessing the link, possibly by brute force and then use that to impersonate the user on Book Bazar platform. 
```

Comment:
For the "There is a risk of..." sentence, I think it would be a bit more clear if it was separated into two sentences. eg.:

...by brute force. When they access the link, they can...